docs/faqs/related_projects.md

```diff
@@ -75,7 +75,7 @@ plane.
 <!-- Named Links -->
 
 [Open Service Broker]: https://www.openservicebrokerapi.org/
-[Kubernetes Service Catalog]: https://kubernetes.io/docs/concepts/extend-kubernetes/service-catalog/
+[Kubernetes Service Catalog]: https://github.com/kubernetes-retired/service-catalog
```

Comment:
FWIW we could completely remove this - I haven't heard anyone talk about it in a while.

(Or not, happy to keep it w/the updated link.)

Reply:
I'll keep it for this PR and remove it in a future update.